src/emulator/controller.ts

```diff
@@ -1,9 +1,12 @@
 import * as _ from "lodash";
 import * as clc from "cli-color";
 import * as fs from "fs";
-import * as pf from "portfinder";
 import * as path from "path";
 
+// No typings exist for this library.
+// tslint:disable-next-line:no-var-requires
+const tcpport = require("tcp-port-used");
```

Comment:
Can you use `@types/tcp-port-used` for the types?

Reply:
D'oh